frontend/exporter/src/types/copied.rs

```diff
@@ -1929,6 +1942,9 @@ pub enum PatKind {
     Deref {
         subpattern: Pat,
     },
+    DerefPattern {
+        subpattern: Pat,
+    },
```

Comment:
Hehehe that was meee https://github.com/rust-lang/rust/pull/122222 :angel:

Reply:
Aaah, that is pretty pretty nice! :superhero::superhero::superhero::superhero: